esm/index.js

```diff
@@ -33,7 +33,7 @@ export class Signal {
   }
 
   /** @returns {T} */
-  then() { return this.value }
+  then(callback) { callback(this.value) }
```

Comment:
can you please rename `callback` to `resolve` instead? thanks!

Reply:
also could you please fix the test [in here](https://github.com/WebReflection/usignal/blob/main/test/test.js#L16-L19)? that should be instead:

```js
    const i = setTimeout(assert, 1000, false, 'promise does not resolve');
    (async () => {
      assert(await signal(3) === 3, 'thenable signal');
      assert(await computed(() => 4) === 4, 'thenable computed');
      clearTimeout(i);
    })();
```